codenames-domain/src/main/java/com/codenames/dto/ErrorDto.java

```diff
@@ -0,0 +1,11 @@
+package com.codenames.dto;
+
+import lombok.*;
```

Comment:
change to single imports

Reply:
Sorry forgot to change the settings in the idea. changed it